docs/opmitize_nydus_image.md

```diff
@@ -1,34 +1,36 @@
-# Optimizing the nydus image
+# Optimize a nydus image
+
+To improve the prefetch hit rate, we can specify a prefetch table when converting an OCI image to a nydus image. The prefetch table is workload related and we should generate the list of accessed files in order when the workload is running. Nydus-snapshotter has implemented an optimizer with Containerd NRI plugin to optimize the nydus image. The optimizer is image format independent and requires NRI 2.0, which is available in containerd 1.7.0 (not yet released). The optimizer subscribe container and sandbox events to watch what files are opened and read, etc. during container application is starting up. This enables nydus image 
```

Comment:
Now the optimizer only subscribes to `StartContainer` event without sandbox events.

Reply:
fixed